awx/main/tasks.py

```diff
@@ -934,6 +934,10 @@ def run(self, pk, **kwargs):
         event_ct = 0
         stdout_handle = None
 
+        # store a record of the venv used at runtime
+        if hasattr(instance, 'custom_virtualenv'):
+            self.update_model(pk, custom_virtualenv=getattr(instance, 'ansible_virtualenv_path', settings.ANSIBLE_VENV_PATH))
```

Comment:
This pattern seems to be already fairly well established, handled generally for all types

https://github.com/ansible/awx/blob/ef1cc5008e5f60ccc0b806a6261d0e43987736dd/awx/main/tasks.py#L971-L972

It made me think "are we using bubblewrap right for inventory update custom venvs", but because the field was already expected (by the generic `run` method), it probably is.